src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,16 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User result = new UserService().findByEmail(email);
+        if (result == null) {
```

Comment:
Better use more meaningful naming instead of `result`. For example you could use `user` variable name